rust/src/delta.rs

```diff
@@ -175,9 +175,14 @@ pub enum DeltaTableError {
     },
     /// Error returned when Vacuum retention period is below the safe threshold
     #[error(
-        "Invalid retention period, retention for Vacuum must be greater than 1 week (168 hours)"
+        "Invalid retention period, retention for Vacuum is configured to be greater than {} milliseconds, got {} milliseconds", .min, .provided
```

Comment:
Since the user provided hours, would it be possible to have the error message report back in hours instead of milliseconds?

Reply:
updated :+1: